experimental/devices/mfrc522/commands/picc.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+package commands
```

Comment:
Keep an empty line between the copyright and the package name, otherwise this becomes the package's documentation.